docs/_events/step_by_step_navigation.md

```diff
@@ -36,5 +36,32 @@ events:
       - name: text
       - name: type
         value: "step by step"
-
+  - name: link clicks
+    description: Triggered when a link within a step by step is clicked, right clicked, shift clicked, control clicked, or windows key/command key clicked.
+    example_url: https://www.gov.uk/learn-to-drive-a-car
+    tracker: link_tracker
+    data:
+    - name: event
+      value: event_data
+    - name: event_data
+      value:
+      - name: event_name
+        value: navigation
+      - name: external
+      - name: index
+        value:
+        - name: index_section
+          value: The section that this link is in. The section in this case refers to the step number.
+        - name: index_link
```

Comment:
I don't think we've added it yet, but will step by steps include `index_section_count`?